modules/dns/variables.tf

```diff
@@ -49,3 +49,15 @@ variable "dns_record_set" {
     error_message = "The specified DNS record type is not valid."
   }
 }
+
+variable "base64_encoded_dns_file" {
+  type        = string
+  description = "The base64 encoded DNS zone file in BIND format that contains the details of the DNS records. For more information about the import format see, [managing DNS records](https://cloud.ibm.com/docs/dns-svcs?topic=dns-svcs-managing-dns-records&interface=ui)."
```

Comment:
[]() -> its .md format. Why to use in terraform file?
For more information about the import format, refer...